types/protobuf.go

```diff
@@ -16,11 +16,10 @@ import (
 // Use strings to distinguish types in ABCI messages
 
 const (
-	ABCIEvidenceTypeDuplicateVote    = "duplicate/vote"
-	ABCIEvidenceTypePhantom          = "phantom"
-	ABCIEvidenceTypeLunatic          = "lunatic"
-	ABCIEvidenceTypePotentialAmnesia = "potential_amnesia"
-	ABCIEvidenceTypeMock             = "mock/evidence"
+	ABCIEvidenceTypeDuplicateVote = "duplicate/vote"
+	ABCIEvidenceTypePhantom       = "phantom"
```

Comment:
I thought we've decided to get rid of phantom evidence, no?

Reply:
Yes we have decided to get rid of it, I just wanted to do it in another PR/commit